package.json

```diff
@@ -227,7 +227,7 @@
     "jest-preset-angular": "~14.1.0",
     "js-yaml": "^4.1.0",
     "jsonc-eslint-parser": "~2.4.0",
-    "jsonpath-plus": "^8.0.0",
+    "jsonpath-plus": "^10.0.0",
```

Comment:
```suggestion
    "jsonpath-plus": "~10.0.0",
```

Reply:
Why do we want to enforce a tilde?

Reply:
The original PR #2286 is with tilde, I think it should be the case of it's replication/cascading.
In general the `~` is more future proof (even if this argument is weak with a lock file, for devDep, in a already released version)